tests/src/tools/ReadyToRun.SuperIlc/Commands/CompileFromCrossgenRspCommand.cs

```diff
@@ -0,0 +1,271 @@
+                cpaotReferencePaths.Add(options.CoreRootOutputPath(CompilerIndex.CPAOT, false));
+                cpaotReferencePaths.AddRange(referencePaths);
+                runners.Add(new CpaotRunner(options, cpaotReferencePaths));
+            }
+
+            if (options.Crossgen)
+            {
+                List<string> crossgenReferencePaths = new List<string>();
+                crossgenReferencePaths.Add(options.CoreRootOutputPath(CompilerIndex.Crossgen, false));
+                crossgenReferencePaths.AddRange(referencePaths);
+                runners.Add(new CrossgenRunner(options, crossgenReferencePaths));
+            }
+
+            return runners;
+        }
+
+        private class CrossgenArguments
+        {
+            public string InputFile;
+            public List<string> ReferencePaths = new List<string>();
+
+            public static CrossgenArguments ParseFromResponseFile(string responseFile)
+            {
+                var arguments = new CrossgenArguments();
+
+                string[] tokenizedArguments = TokenizeArguments(responseFile);
+                
+                for (int i = 0; i < tokenizedArguments.Length; i++)
+                {
+                    string arg = tokenizedArguments[i];
+                    if (MatchParameter("in", arg))
+                    {
+                        arguments.InputFile = tokenizedArguments[i + 1];
+                    }
+                    else if (MatchParameter("App_Paths", arg) || MatchParameter("Platform_Assemblies_Paths", arg))
+                    {
+                        string appPaths = tokenizedArguments[i + 1];
+                        arguments.ReferencePaths.AddRange(appPaths.Split(';').TakeWhile(x => !string.IsNullOrWhiteSpace(x)));
+                        ++i;
+                    }
+                    else if (MatchParameter("verbose", arg)
+                        || MatchParameter("readytorun", arg))
+                    {
+                        // Skip unparameterized switches
+                        continue;
+                    }
+                    else if (MatchParameter("jitpath", arg))
+                    {
+                        // Skip switches with one parameter
+                        ++i;
+                        continue;
+                    }
+                    else if (!IsSwitch(arg))
+                    {
+                        Debug.Assert(arguments.InputFile == null);
+                        arguments.InputFile = arg;
+                    }
+                }
+
+                return arguments;
+            }
+
+            public CrossgenArguments ReplacePaths(Dictionary<string, string> replacementPaths)
+            {
+                foreach (var replacePath in replacementPaths)
+                {
+                    if (InputFile.StartsWith(replacePath.Key, Environment.OSVersion.Platform == PlatformID.Win32NT, CultureInfo.CurrentCulture))
```

Comment:
Is "CultureInfo.CurrentCulture" going to guarantee case-insensitive path equality on Windows?

Reply:
`CurrentCulture` is what's used by default. The second argument with the platform compare is what ensures path equality will be case sensitive on platforms where that's needed. I named the arguments and passed null for `culture` to be explicit that we just want what string does by default.